catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -103,6 +107,7 @@
  * Federated site that talks via OpenSearch to the DDF platform. Communication is usually performed
  * via https which requires a keystore and trust store to be provided.
  */
+@SuppressWarnings("WeakerAccess")
```

Comment:
✏️ Usually when we suppress warnings we add a comment for why e.g.
`@SuppressWarnings("WeakerAccess" /* Needed by blueprint. */)` or something of that nature.

Reply:
removed it